icons/table-columns-split.json

```
@@ -0,0 +1,16 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "chessurisme"
+  ],
+  "tags": [
+    "spreadsheet",
+    "grid",
+    "table",
+    "column"
```

These tags are not necessary, as they are part of the name, but some synonyms for the action itself could be useful:
```suggestion
    "cut",
    "break",
    "divide",
    "separate",
    "segment"
```